src/test/groovy/com/github/vkorobkov/jfixtures/loader/FixtureTest.groovy

```diff
@@ -120,4 +120,71 @@ class FixtureTest extends Specification {
         then:
         merged.name == "users"
     }
+
+    def "#mergeFixtures puts fixtures for different tables in sequence"() {
+        given:
+        def users = new Fixture("users", [
+            new FixtureRow("Vlad",  [name: "Vlad", age: 30]),
+            new FixtureRow("Burns", [name: "Mr Burns", age: 130])
+        ])
+        def roles = new Fixture("roles", [
+            new FixtureRow("user",  [type: "user", readAccess: true, writeAccess: false]),
+            new FixtureRow("admin", [type: "admin", readAccess: true, writeAccess: true])
+        ])
+
+        when:
+        def merged = Fixture.mergeFixtures([users, roles])
+
+        then:
+        merged.size() == 2
```

Comment:
can we aggregate this with "with"?

Reply:
and 2 below

Reply:
I tried, by access by index does not work with `with`:
```groovy
with(merged) {
            [0].name == "users"
            [0].rows.toList() == users1.rows + users2.rows
        }
```

Produces failure:
```
Condition failed with Exception:

[0].name == "users"
    |
    groovy.lang.MissingPropertyException: Exception evaluating property 'name' for java.util.ArrayList, Reason: groovy.lang.MissingPropertyException: No such property: name for class: java.lang.Integer
```